### ISSUE TITLE
Error with BookmarkManager.async_create

### ISSUE THREAD

**Issue by kinduff** [bachya/linkding-cli#36]
**Describe the bug**
There is an exception when trying to create a bookmark. Read operations are working as expected, so API connection is not the issue.

**To Reproduce**

1. Run `linkding --verbose bookmarks create -a https://github.com`
2. Press enter
3. See error

**Expected behavior**
Create a new bookmark.

**Additional context**

Output
```
$ linkding --verbose bookmarks create -a https://github.com
2022-11-24 16:27:29,855 | linkding_cli | DEBUG | Command: bookmarks
2022-11-24 16:27:29,855 | linkding_cli | DEBUG | Arguments: []
2022-11-24 16:27:29,855 | linkding_cli | DEBUG | Options: {'verbose': True, 'config': None, 'token': '[REDACTED]', 'url': 'https://[REDACTED]'}
2022-11-24 16:27:29,855 | linkding_cli | DEBUG | Loaded Config: <Config token=[REDACTED] url=https://[REDACTED] verbose=True>
2022-11-24 16:27:29,856 | linkding_cli | ERROR | BookmarkManager.async_create() got an unexpected keyword argument 'archived'
2022-11-24 16:27:29,857 | linkding_cli | DEBUG |   File "/home/kinduff/.local/lib/python3.10/site-packages/linkding_cli/helpers/logging.py", line 79, in wrapper
    return cast(dict[str, Any], func(*args, **kwargs))
  File "/home/kinduff/.local/lib/python3.10/site-packages/linkding_cli/commands/bookmark.py", line 109, in create
    data = asyncio.run(ctx.obj.client.bookmarks.async_create(url, **payload))
```

- Python 3.10.6
- pip 22.0.2 from /usr/lib/python3/dist-packages/pip (python 3.10)
- Linkding 1.15.1 (latest)

**Comment by bachya** [comment]
Just published [2022.11.0](https://github.com/bachya/linkding-cli/releases/tag/2022.11.0), which contains this fix. Thanks for your help!

**Comment by kinduff** [comment]
Thanks a lot @bachya! Great response and very fast. I appreciate the effort!
